FlickrViewer/Podfile

```diff
@@ -0,0 +1,13 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'FlickrViewer' do
+  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for FlickrViewer
+pod 'SDWebImage','~>4.0'
```

Comment:
for all pods need to use specific version like
 pod 'SDWebImage', '4.0'
to prevent unintended pod update conflicts